include/rocksdb/convenience.h

```diff
@@ -26,6 +26,10 @@ struct Options;
 // of the serialization (e.g. delimiter), and how to compare
 // options (sanity_level).
 struct ConfigOptions {
+  // Returns a ConfigOptions suitable for nested/embedded
+  // options.
+  ConfigOptions Embedded() const;
```

Comment:
The usage of this function is unclear to me. Can you explain? Based on its implementation
` ConfigOptions embedded = *this;
  embedded.delimiter = ";";
  return embedded;`
Looks like we do not need this function?